filebeat/beater/filebeat.go

```diff
@@ -173,15 +173,15 @@ func (fb *Filebeat) setupPipelineLoaderCallback(b *beat.Beat) error {
 		}
 
 		// When running the subcommand setup, configuration from modules.d directories
-		// have to be loaded using cfg.Reloader. Otherwise those configurations are skipped.
+		// have to be loaded using cfg.Reloader. Otherwise, those configurations are skipped.
 		pipelineLoaderFactory := newPipelineLoaderFactory(b.Config.Output.Config())
 		enableAllFilesets, _ := b.BeatConfig.Bool("config.modules.enable_all_filesets", -1)
 		modulesFactory := fileset.NewSetupFactory(b.Info, pipelineLoaderFactory, enableAllFilesets)
 		if fb.config.ConfigModules.Enabled() {
 			if enableAllFilesets {
-				//All module configs need to be loaded to enable all the filesets
-				//contained in the modules.  The default glob just loads the enabled
-				//ones.  Switching the glob pattern from *.yml to * achieves this.
+				// All module configs need to be loaded to enable all the filesets
```

Comment:
We explicitly disable modules (and filesets) when running under agent: https://github.com/elastic/elastic-agent/blob/7f7faf365aeb303c6e4bb557f3f4b7f4de4025e7/specs/filebeat.spec.yml#L41

This functionality is replaced with integrations.